inc/components/class-google-analytics.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Class file for the Google Analytics component.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving\Component;
+
+/**
+ * Defines the Google Analytics component.
+ */
+class Google_Analytics extends Component {
+
+	/**
+	 * Unique component slug.
+	 *
+	 * @var string
+	 */
+	public $name = 'google-analytics';
+
+	/**
+	 * Initalize this component.
+	 */
+	public function __construct() {
+		parent::__construct();
```

Comment:
I believe the parent constructor takes a few arguments, don't you want to pass those through?

Reply:
Good point. I'll follow up in a subsequent PR.